crates/federate/src/inboxes.rs

```diff
@@ -0,0 +1,149 @@
+use crate::util::LEMMY_TEST_FAST_FEDERATION;
```

Comment:
as a note, i've not made any changes to this code, just moved it into a separate struct

Reply:
update: the code now has minor (non-functional) modifications to allow testing it